packages/vue/src/components/molecules/SfModal/SfModal.js

```diff
@@ -4,18 +4,18 @@ export default {
   name: "SfModal",
   props: {
     /**
-     * Cross closing modal button
+     * Visibility of the modal
      */
-    cross: {
+    value: {
       type: Boolean,
-      default: true
+      default: false
     },
     /**
-     * Visibility of the modal
+     * Cross closing modal button
      */
-    visible: {
+    cross: {
```

Comment:
I don't think `value` is a good property name here, instead of changing property name, we can set [custom `v-model` prop and event](https://scotch.io/tutorials/add-v-model-support-to-custom-vuejs-component#toc-customizing-v-model-prop-and-event) like that:

```js
model: {
  prop: 'visible',
  event: 'toggle'
}
```

or, if you want to also support `visible.sync`:

```js
model: {
  prop: 'visible',
  event: 'update:visible'
}
```

wdyt?

Reply:
https://github.com/DivanteLtd/storefront-ui/pull/459